scripts/exgfs_atmos_awips_20km_1p0deg.sh

```diff
@@ -205,21 +205,21 @@ for GRID in conus ak prico pac 003; do
       # Post Files to ${COMOUT_ATMOS_WMO}
       ##############################
 
-      mv "grib2.awpgfs_20km_${GRID}_f${fcsthrs}" \
-         "${COMOUT_ATMOS_WMO}/grib2.awpgfs_20km_${GRID}_f${fcsthrs}"
+      mv "grib2.awpgfs_20km_${GRID}_f${fcsthr}" \
```

Comment:
should this be `cpfs`?  Why `mv`?

Reply:
@ChristopherHill-NOAA Are there any issues if we change this `mv` to a copy (`cpfs`)?

Reply:
> @ChristopherHill-NOAA Are there any issues if we change this `mv` to a copy (`cpfs`)?

@KateFriedman-NOAA I believe it could only help to ensure that `grib2.awpgfs_20km_${GRID}_f${fcsthr}` has successfully moved to `${COMOUT_ATMOS_WMO}` as a complete file within the allotted runtime, and to append the file with `.cptmp` if this is not the case.

Reply:
Thanks for the feedback @ChristopherHill-NOAA ! I have updated the `mv`s in the script to `cpfs` @ e8a78dc83.